sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -851,6 +855,19 @@ ProgramManager::build(ProgramPtr Program, const ContextImplPtr Context,
   return Program;
 }
 
+static ProgramManager::KernelArgMask
+createKernelArgMask(const std::vector<unsigned char> &Bytes) {
+  int SizeInBits = Bytes[0];
```

Comment:
SizeInBits occupies first 8 bytes, not one.

Reply:
Fixed in the latest commit